composer.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "WOVNio/WOVN.php",
-  "type": "Library",
+  "type": "library",
```

Comment:
The docker image of circleci/php has changed recently.
Maybe composer was upgrated and it starts to cause the following error.
So, I changed to use lowercase.
```
  [Composer\Json\JsonValidationException]                     
  "./composer.json" does not match the expected JSON schema:  
   - type : Does not match the regex pattern ^[a-z0-9-]+$  
```